docs/config/README.md

```diff
@@ -650,6 +651,41 @@ By default the module will be shown if any of the following conditions are met:
 format = "via [🔰 $version](bold red) "
 ```
 
+## Deno
+
+The `deno` module shows you your currently installed version of Deno.
+By default the module will be shown if any of the following conditions are met:
+- The current directory contains a `mod.ts`, `mod.js`, `deps.ts` or `deps.js` file
+
+### Options
+
+| Option              | Default                                           | Description                                     |
+| ------------------- | ------------------------------------------------- | ----------------------------------------------- |
+| `format`            | `"via [$symbol($version )]($style)"`              | The format for the module.                      |
+| `symbol`            | `"🦕 "`                                           | A format string representing the symbol of Dart |
```

Comment:
Emojis need to have two instead of one space after them to avoid rendering bugs:
```suggestion
| `symbol`            | `"🦕  "`                                           | A format string representing the symbol of Dart |
```